modules/displayLists.js

```diff
@@ -0,0 +1,80 @@
+import removeListitem from './removeListitem.js';
+import editList from './editList.js';
+import markComplete from './markComplete.js';
+import markUnComplete from './markUnComplete.js';
+
+export default function displayLists(taskList) {
```

Comment:
- [ ]  Kindly check my comment on the use of es6 on the addToList file which is very detailed 👍